src/Utilities/matrix_of_constraints.jl

```diff
@@ -352,6 +352,7 @@ function _add_constraint(
     func::F,
     set::S,
 ) where {F<:MOI.AbstractFunction,S<:MOI.AbstractSet}
+    func = MOI.Utilities.canonical(func)
```

Comment:
Why call canonical twice? (Here and `add_constraint`?)

Reply:
Oops, this one shouldn't be